### PR TITLE
DRAFT: Added flag to enable per job debug logging INT-1076 INT-3615

### PR DESCRIPTION
**Pull request by vladt**
This PR adds a flag that allows users to enable debug logging for the nexus jenkins plugin on a per job basis.
For jobs:
image
For pipelines:
image

https://issues.sonatype.org/browse/INT-3615
https://jenkins.ci.sonatype.dev/job/integrations/job/jenkins/job/nexus-platform-plugin-feature/job/INT-3615_DebugLoggingFlag/lastBuild/

I'll add tests if this solution is accepted.

### CODE REVIEW

**Comment by eduard-tita** [comment]
It looks good to me.

File: src/main/java/org/sonatype/nexus/ci/util/LoggerBridge.groovy [hunk 1]
```diff
@@ -52,12 +52,16 @@ final class LoggerBridge
 
   @Override
    boolean isTraceEnabled() {
-    return DEBUG
+    return debugEnabled
```
Comment (rjackson64840) [inline src/main/java/org/sonatype/nexus/ci/util/LoggerBridge.groovy:55]: Why is TRACE tied to the same value?

**Review by rjackson64840 — APPROVED** [review]
+1 : just one question

File: src/main/java/org/sonatype/nexus/ci/util/LoggerBridge.groovy (hunk 1 above)
Comment (vladt) [reply src/main/java/org/sonatype/nexus/ci/util/LoggerBridge.groovy:55]: > Why is TRACE tied to the same value?

I don't know.

**Pull request closed without merging**